### PR TITLE
Fix left over "script" in default config when using perform action

### PR DESCRIPTION
**Pull request by silamon**
## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

## Summary by CodeRabbit

- **New Features**
	- Updated configuration for service actions to focus on specific actions instead of generic service identifiers, enhancing clarity in usage.
  
- **Bug Fixes**
	- Adjusted the return structure of default configuration to prevent potential miscommunication with other components relying on this configuration.

### CODE REVIEW

**Review by frenck — APPROVED** [review]
Thanks, @silamon 👍 

../Frenck

File: src/panels/config/automation/action/types/ha-automation-action-service.ts [hunk 1]
```diff
@@ -53,7 +53,7 @@ export class HaServiceAction extends LitElement implements ActionElement {
   );
 
   public static get defaultConfig() {
-    return { service: "", data: {} };
+    return { action: "", data: {} };
```
[the remark this answers is not in the document]
Comment (silamon) [reply src/panels/config/automation/action/types/ha-automation-action-service.ts:56]: I'm not sure how unit tests would work, but if you could add the typing for this method and any other `public static get defaultConfig()` existing in the code , that would be great.  Open a new pull request and I'll review.  Good luck.

[the remark this answers is not in the document]
Comment (silamon) [reply src/panels/config/automation/action/types/ha-automation-action-service.ts:56]: Go ahead and open a pull request. :D

[the remark this answers is not in the document]
Comment (silamon) [reply src/panels/config/automation/action/types/ha-automation-action-service.ts:56]: Open an issue for further following up.

**Pull request merged**